### DISCUSSION TITLE
Remove semantic commit prefixes from Dependency Dashboard list items

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#24898] [Suggest an Idea]
**What would you like Renovate to be able to do?**

The Renovate Dependency Dashboard can have long lines for dependency updates:

```markdown
- [ ] chore(deps): update jest monorepo (`@jest/globals`, `@jest/reporters`, `@jest/test-result`, `jest, jest-circus`)
```

I'd like to trim down the text so that we have room to add a Merge Confidence badge and emoji (see #10232).

**Did you already have any implementation ideas?**

@rarkins liked my idea to remove the "semantic commit prefixes" from the line items.

Maybe we can make a template out of this feature, so that end-users can choose which display format they like best: trimmed down, or full-fat.

**Input:**

```markdown
- [ ] chore(deps): update jest monorepo (`@jest/globals`, `@jest/reporters`, `@jest/test-result`, `jest, jest-circus`)
```

**Output:**

```markdown
- [ ] update jest monorepo (`@jest/globals`, `@jest/reporters`, `@jest/test-result`, `jest, jest-circus`)
```

**Comment by viceice** [comment]
I also like to drop `update` word. 🙃

**Comment by HonkingGoose** [comment]
> I also like to drop `update` word. 🙃

That's even shorter! 🚀 

We can get away with dropping the word `update`, because the "Dependency Dashboard" explains that the list items are dependency updates. 😉 

> This issue contains a list of Renovate updates and their statuses.

---

Let's revise the requirements then. 😉 

**Prior line item:**

```markdown
- [ ] chore(deps): update jest monorepo (`@jest/globals`, `@jest/reporters`, `@jest/test-result`, `jest, jest-circus`)
```

**New line item:**

```markdown
- [ ] jest monorepo (`@jest/globals`, `@jest/reporters`, `@jest/test-result`, `jest, jest-circus`)
```
